src/OpenCV_Mat.jl

```diff
@@ -195,25 +195,25 @@ function pixset(img, row::Int, col::Int, value)
   # RGB images (value::  std::vector<double>)
   elseif (cvtypeval(img) == CV_8SC3)
     cppvec = tostdvec(value)
```

Comment:
To reproduce the error : 
In `test/runtests.jl`, import the following 
```julia
using OpenCV
using Images
using Base.Test
using FileIO
``` 
In `test/jl/tests.jl`, add the following test function 
```julia
function test6()
   println("test 6: Conversion of images from Images.jl to OpenCV Mat")
   filename = joinpath(Pkg.dir("OpenCV"), "./test/images/mandrill.jpg")
   image = FileIO.load(filename)
   dst = convertToMat(image)
   println("Image converted to Mat successfully")
end

test6()
```
`src/OpenCV_Mat.jl` as in master branch.
In `src/OpenCV_ImageSupport.jl`, the following function definition : 

```julia
function convertToMat(image)
    img = permuteddimsview(channelview(image), (2,3,1))
    cd = Base.size(channelview(image))[1] > 3 ? 1 : 3
    _rows = Base.size(image, 1)
    _cols = Base.size(image, 2)
    if (typeof(img[1,1,1].i) == UInt8)
       if (cd < 3); mat = Mat(512, 512, CV_8UC1); end
       if (cd == 3); mat = Mat(512, 512, CV_8UC3); end
    elseif (typeof(img[1,1,1].i) == Float32)
       if (cd < 3); mat = Mat(512, 512, CV_32FC1); end
       if (cd == 3); mat = Mat(512, 512, CV_32FC3); end
    elseif (typeof(img[1,1,1].i) == Float64)
       if (cd < 3); mat = Mat(512, 512, CV_64FC1); end
       if (cd == 3); mat = Mat(512, 512, CV_64FC3); end
    else
       throw(ArgumentError("Pixel format not supported!"))
    end

    if (cd < 3)   # grayscale or binary
        for j = 1:_rows     # index row first (Mat is row-major order)
            for k =1:_cols  # index column second
                # slow algorithm  - will try to use pointer method (C++)!
                pixset(mat, k, j, float(img[k,j,1].i))
            end
        end
    end

   if (cd == 3)   # color (RGB) image
        for j = 1:512     # index row first (Mat is row-major order)
            for k =1:512  # index column second
                colorvec = tostdvec([float(img[k,j,1].i),float(img[k,j,2].i),float(img[k,j,3].i)])
                pixset(mat, k-1, j-1, colorvec)   # -1 to have 0-indexing per C++
            end
        end
    end

    return(mat)
end
```